news/_posts/public-oss-demo/2022-07-06-public-demo-5-2022.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "PrestaShop Project Public Demo - July 2022 session"
+subtitle: "We held the fifth session of this year open source project's public demo"
+date:   2022-07-06 22:00
+authors: [ PrestaShop ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+The project's fifth session of this year edition of our public demo was held and [streamed live](https://www.youtube.com/watch?v=-JzJTygO_bg) on July 6th, 2022.
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=-JzJTygO_bg) Beginning of the session, the introduction
+- [02:16](https://www.youtube.com/watch?v=-JzJTygO_bg&t=136s) Project news: Welcome new committers to the project
+- [03:32](https://www.youtube.com/watch?v=-JzJTygO_bg&t=212s) Project news: PrestaShop 8 Feature-Freeze
+- [07:54](https://www.youtube.com/watch?v=-JzJTygO_bg&t=474s) Project news: Product Discovery for PrestaShop 8.1 and above
+- [11:41](https://www.youtube.com/watch?v=-JzJTygO_bg&t=701s) Demo: New password policy [@matks](https://github.com/matks)
```

Comment:
Note: you can write `@matks` instead of `[@matks](https://github.com/matks)` it works 😄 I discovered this randomly

Reply:
Nice! Thanks, it'll save some time :D